sources/canadian-words.10

```diff
@@ -1,6 +1,7 @@
 among
 analog
 apologize
+backwards
```

Comment:
@jacksonrayhamilton "backwards" is already part of `english-words.10`—isn't this file supposed to be strictly for the Canadian-specific words?

Reply:
@jacksonrayhamilton Never mind—I see that the latest SCOWL version you updated to has "backwards" removed from `english-words.10`. Resolving this thread.